### DISCUSSION TITLE
Multiple domains section

### DISCUSSION THREAD

**Discussion by odoo-sh** [postalserver/postal#1673] [Help with using Postal] [answered]
Hi,

Can someone explain what is the difference between "Add new domain" under organization and Under Mail-servers? 
Is there any specific purpose for creating domains at two different places? 

1. https://postal.com/org/mycomany/domains
2. https://postal.com/org/mycomany/servers/production/domains

Thanks,

**Comment by willpower232** [comment, marked as answer]
The domains under the organisation (number 1 on your list) apply to all servers under the organisation so you could add your company domain to the organisation and break down message handling with separate servers depending on how your use case is structured.

**Reply by odoo-sh** [reply]
Thanks. That makes sense.
